modules/ebi-metagenomics/hmmer/hmmsearch/meta.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hmmer_hmmsearch"
+description: search for sequences against a HMM profile database
+keywords:
+  - hmm
+  - hmmer
+  - sequence search
+  - hmmsearch
+tools:
+  - "hmmer":
+      description: "Biosequence analysis using profile hidden Markov models"
+      homepage: http://hmmer.org/
+      documentation: http://hmmer.org/documentation.html
+      tool_dev_url: https://github.com/EddyRivasLab/hmmer
+      doi: "10.1371/journal.pcbi.1002195"
+      licence: ["BSD"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - fasta:
+      type: file
+      description: FASTA file containing amino acid sequences to be searched for in profile
+      pattern: "*.fasta"
+  - hmm_db:
+      type: file
+      description: HMM profile database
```

Comment:
I would refrain from using the word database. Correct me if I'm wrong, but hmmsearch takes a single HMM profile every time to fish hits from a sequence db. Just "HMM profile" should work the best here I think.